service/controller/resource/tccpnoutputs/create.go

```diff
@@ -83,6 +84,16 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		cc.Status.TenantCluster.OperatorVersion = v
 	}
 
+	{
+		v, err := cloudFormation.GetOutputValue(outputs, ReleaseVersionKey)
```

Comment:
We use the output value from the CF to check the components in case we have a new release version

Reply:
This is for the tccpn part